app/models/features_plan.rb

```diff
@@ -6,9 +6,6 @@ class FeaturesPlan < ApplicationRecord
 
   validate :feature_scope_matches_plan_class?
 
-  # TODO: enable this on rails 3.1
-  # validates :plan_id, :plan_type, :feature_id, :presence => true
```

Comment:
The validation itself seems correct for me, but it's commented out so no effect currently, and since the comment says "enable on rails 3.1" and we are on rails 5, I considered that TODO is expired at this point.

Do you think we should keep the comment? or even uncomment the validation?